src/Integration/LocalServices/ErrorListInfoBarController.cs

```diff
@@ -112,17 +119,6 @@ public void Refresh()
             private set;
         }
 
-        [Conditional("DEBUG")]
```

Comment:
No longer necessary. I changed the code from asserting to throwing if it is called on the wrong thread. There are only a couple of callers, so we'll soon find out if I missed one :-)

Reply:
How to introduce fun in the workspace!